spigot/src/main/java/io/github/retrooper/packetevents/util/SpigotConversionUtil.java

```diff
@@ -109,4 +118,16 @@ public static ItemStack fromBukkitItemStack(org.bukkit.inventory.ItemStack itemS
     public static org.bukkit.inventory.ItemStack toBukkitItemStack(ItemStack itemStack) {
         return SpigotReflectionUtil.encodeBukkitItemStack(itemStack);
     }
+
+    public static Dimension fromBukkitWorld(World world) {
+        boolean v1_19 = PacketEvents.getAPI().getServerManager().getVersion().isNewerThanOrEquals(ServerVersion.V_1_19);
+        boolean v1_16_2 = PacketEvents.getAPI().getServerManager().getVersion().isNewerThanOrEquals(ServerVersion.V_1_16_2);
+        if (v1_19 || !v1_16_2) {
```

Comment:
Isn't this always true?

Reply:
Only versions 1.17 and 1.18 changed the dimension serialization to nbt